src/05-derivatives/02-common-data-types.md

```diff
@@ -0,0 +1,193 @@
+# Common data types and metadata
+
+## Common file level metadata fields
+
+Each derivative data file SHOULD be described by a JSON file provided as a sidecar
+or higher up in the hierarchy of the derived dataset (according to the
+[Inheritance Principle](../02-common-principles.md#the-inheritance-principle))
+unless a particular derivative includes REQUIRED metadata fields, in which case a
+JSON file is also REQUIRED.
+Each derivative type defines their own set of fields, but all of them
+share the following (non-required) ones:
+
+| **Key name** | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Description  | RECOMMENDED. Free-form natural language description of the nature of the file.                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| Sources      | OPTIONAL. A list of files with the paths specified relative to dataset root; these files were directly used in the creation of this derivative data file. For example, if a derivative A is used in the creation of another derivative B, which is in turn used to generate C in a chain of A->B->C, C should only list B in `Sources`, and B should only list A in `Sources`. However, in case both X and Y are directly used in the creation of Z, then Z should list X and Y in `Sources`, regardless of whether X was used to generate Y. |
```

Comment:
Accepted first suggestion in https://github.com/bids-standard/bids-specification/pull/488#discussion_r433285725.